### PR TITLE
geijn2022reproducibility

### PR DESCRIPTION
**Pull request by VicK9**
## Project Name
geijn2022reproducibility

## Compiled PDF
The compiled PDF has errors.
[Compiled Pdf](2021/geijn2022reproducibility/journal/article.pdf)

## Metadata Checklist

Ensure that you have added the following metadata information (put an x to the checkbox items) in `geijn2022reproducibility/journal/metadata.yaml`.

- [x] `title` field exists and it starts with `[Re] `.
- [x] `authors` contain author information with affiliations.
- [x] [ORCID id](https://orcid.org/) is provided for authors.
- [x] One contact author is added using `*`.
- [x] Github `url` is provided in `code` section.
- [x] [SWH identifier](https://archive.softwareheritage.org/save/) is provided in `swh`.
- [x] `replication` section is populated with the necessary information about the paper you reproduced.
- [x] `abstract` field is populated with your paper abstract (including double quotes)
- [x] `domain` should contain "ML Reproducibility Challenge 2021"
- [x] `review` section contains the url to the OpenReview forum of your paper.
- [x] Fields `contributors`, `dates`, `article` and `journal` are kept blank, as those will be populated by the Area Chairs / ReScience editors.

## Build Errors

The compiled Pdf has some errors that we could not fix after following the steps.
1) Editors info overlap with text
2) References and citations do not work

We provide the link to the buid.log we get: [Pastebin](https://pastebin.com/7Ax9xgNr)

### CODE REVIEW

**Comment by VicK9** [comment]
The compiled Pdf has some errors that we could not fix after following the steps.

Editors info overlap with text
References and citations do not work

------------
Run number 1 of rule 'pdflatex'
------------
------------
Running 'xelatex -interaction=nonstopmode  -recorder  "article.tex"'
------------
Latexmk: References changed.
Latexmk: Non-existent bbl file 'article.bbl in line'
 No file article.bbl.
Latexmk: References changed.
Latexmk: References changed.
Latexmk: Log file says output to 'article.pdf'
Latexmk: List of undefined refs and citations:
  Citation '11goyal2019counterfactual' on page 3 undefined on input line 41
  Citation '17kim2018interpretability' on page 3 undefined on input line 39
  Citation '19mothilal2020explaining' on page 3 undefined on input line 41
  Citation '33goyal2019counterfactual' on page 3 undefined on input line 41
  Citation '35wu2021stylespace' on page 3 undefined on input line 43
  Citation 'Karras2020AnalyzingAI' on page 3 undefined on input line 43
  Citation 'Karras2020AnalyzingAI' on page 6 undefined on input line 102
  Citation 'Karras2020AnalyzingAI' on page 6 undefined on input line 113
  Citation 'Lang2021ExplainingIS' on page 1 undefined on input line 5
  Citation 'Lang2021ExplainingIS' on page 3 undefined on input line 44
  Citation 'MILLER20191' on page 3 undefined on input line 38
  Citation 'NIPS2017_892c3b1c' on page 6 undefined on input line 113
  Citation 'deng2012mnist' on page 7 undefined on input line 121
  Citation 'howard2017mobilenets' on page 5 undefined on input line 99
  Citation 'karras2019stylebased' on page 5 undefined on input line 91
  Citation 'karras2019stylebased' on page 5 undefined on input line 95
  Citation 'karras2019stylebased' on page 6 undefined on input line 119
  Citation 'schutte2021using' on page 3 undefined on input line 44
  Reference `app:A' on page 7 undefined on input line 154
  Reference `app:B' on page 7 undefined on input line 158
  Reference `app:MNIST_Reconstruct' on page 10 undefined on input line 266
  Reference `app:MNIST_Reconstruct' on page 9 undefined on input line 237
  Reference `app:classificationstudy' on page 9 undefined on input line 228
  Reference `app:hyperparameters' on page 7 undefined on input line 126
  Reference `fig:MNIST_Reconstruct' on page 10 undefined on input line 266
  Reference `fig:PyTorch vs TensorFlow classifier' on page 7 undefined on input line 154
  Reference `fig:att3' on page 8 undefined on input line 162
  Reference `fig:independent method' on page 10 undefined on input line 262
  Reference `fig:independent method' on page 10 undefined on input line 266
  Reference `fig:independent method' on page 7 undefined on input line 160
  Reference `fig:top 4 attrs ours' on page 10 undefined on input line 266
  Reference `fig:top 4 attrs ours' on page 7 undefined on input line 158
  Reference `fig:top 4 attrs ours' on page 8 undefined on input line 203
  Reference `fig:top 4 attrs theirs' on page 7 undefined on input line 158
  Reference `itm:claim1' on page 5 undefined on input line 91
  Reference `itm:claim1' on page 7 undefined on input line 154
  Reference `itm:claim1' on page 9 undefined on input line 241
  Reference `itm:claim2' on page 10 undefined on input line 258
  Reference `itm:claim2' on page 8 undefined on input line 162
  Reference `itm:claim3' on page 5 undefined on input line 91
  Reference `itm:claim3' on page 8 undefined on input line 221
  Reference `itm:claim3' on page 9 undefined on input line 243
  Reference `sec:hyperparameters' on page 6 undefined on input line 111
  Reference `sec:results' on page 9 undefined on input line 248
  Reference `tab:hyperparameters' on page 7 undefined on input line 126
  Reference `tab:top3' on page 9 undefined on input line 228
  Reference `tab:user study table' on page 9 undefined on input line 226
------------
Running 'make "article.bbl"'
------------
make[1]: *** No rule to make target 'article.bbl'.  Stop.
Latexmk: Summary of warnings from last run of *latex:
  Latex failed to resolve 30 reference(s)
  =====Latex reported missing or unavailable character(s).
=====See log file for details.
  Latex failed to resolve 18 citation(s)
Collected error summary (may duplicate other messages):
  pdflatex: Command for 'pdflatex' gave return code 1
      Refer to 'article.log' for details
Latexmk: Use the -f option to force complete processing,
 unless error was exceeding maximum runs, or warnings treated as errors.
make: *** [Makefile:30: article.pdf] Error 12
Build complete
rm: cannot remove 'article.bbl': No such file or directory
rm: cannot remove 'article.blg': No such file or directory

File: 2021/geijn2022reproducibility/openreview/packages.tex [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    % \usepackage[preprint]{neurips_2019}
+
+% to compile a camera-ready version, add the [final] option, e.g.:
+% \usepackage[final]{neurips_2019}
+
+% to avoid loading the natbib package, add option nonatbib:
+%     \usepackage[nonatbib]{neurips_2019}
+
+\usepackage[utf8]{inputenc} % allow utf-8 input
+\usepackage[T1]{fontenc}    % use 8-bit T1 fonts
```
Comment (koustuvsinha) [inline 2021/geijn2022reproducibility/openreview/packages.tex:22]: Please remove this import (`fontenc`) as it messes up with ReScience (details in Readme).

File: 2021/geijn2022reproducibility/openreview/packages.tex [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+\usepackage{hyperref}       % hyperlinks
+\usepackage{url}            % simple URL typesetting
+\usepackage{booktabs}       % professional-quality tables
+\usepackage{amsfonts}       % blackboard math symbols
+\usepackage{nicefrac}       % compact symbols for 1/2, etc.
+\usepackage{microtype}      % microtypography
+
+\usepackage[dvipsnames]{xcolor}
```
Comment (koustuvsinha) [inline 2021/geijn2022reproducibility/openreview/packages.tex:30]: Please also remove this import (`xcolor`) as it conflicts with ReScience.cls.

File: 2021/geijn2022reproducibility/openreview/packages.tex [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+\newif{\ifhidecomments}
+\usepackage{graphicx}
+\usepackage{booktabs}
+\usepackage{float}
+\usepackage{longtable}
+\usepackage{todonotes} 
+\usepackage{setspace}
+\usepackage{titlesec}
+\usepackage{enumitem}
+\usepackage{xcolor}
```
Comment (koustuvsinha) [inline 2021/geijn2022reproducibility/openreview/packages.tex:41]: Another import of xcolor which needs to be removed.

**Pull request merged**